libs/model-assessment/src/lib/ModelAssessmentDashboard/Controls/IndividualFeatureImportanceView/TabularLocalImportancePlots.tsx

```diff
@@ -75,7 +75,14 @@ export class TabularLocalImportancePlots extends React.Component<ITabularLocalIm
           key,
           text: meta.abbridgedLabel
         };
-      });
+      })
+      .filter(
+        (item) =>
+          !this.context.jointDataset.datasetMetaData?.featureMetaData?.dropped_features?.includes(
```

Comment:
nit: same thing here as above, maybe make this a set before calling filter + includes?